.github/workflows/pull_request-test-go.yaml

```diff
@@ -9,6 +9,12 @@ jobs:
     runs-on: ubuntu-latest
     env:
       GOPATH: /home/runner/work/ARO-RP/go
+      FEATURE_OPERATIONS: ${{ matrix.FEATURE_OPERATIONS }}
+      FEATURE_METRICS: ${{ matrix.FEATURE_METRICS }}
+    strategy:
+      matrix:
+       FEATURE_OPERATIONS: ["true", ""]
```

Comment:
This looks a bit strange in the UI. Will try to see if I can tune the name of the job.

![image](https://user-images.githubusercontent.com/509198/72455947-a1221980-37bb-11ea-8c13-bddf7e10cdc5.png)

Reply:
Ok, this looks better:

![image](https://user-images.githubusercontent.com/509198/72521989-85b71d00-3854-11ea-89a2-b8945c252691.png)